onnxruntime/core/optimizer/pad_fusion.cc

```diff
@@ -54,6 +36,45 @@ bool PadFusion::SatisfyCondition(const Graph& graph, const Node& node, const log
     return false;
   }
 
+  return true;
+}
+
+void Update_Pad_Attribute(Node& child_node, const std::vector<int64_t>& pads_values, const uint32_t pads_size) {
```

Comment:
```suggestion
void UpdatePaddingAttributes(Node& child_node, const std::vector<int64_t>& pads_values, const uint32_t pads_size) {
```
I've never seen ORT use Camel_Case_And_Snake_Case for functions.